smart-send-logistics/includes/class-ss-shipping-wc-order.php

```diff
@@ -85,14 +88,17 @@ public function add_meta_box()
             $order_id = $post->ID;
 
             $ss_shipping_method_id = $this->get_smart_send_method_id($order_id);
+            $screen = wc_get_container()->get( CustomOrdersTableController::class )->custom_orders_table_usage_is_enabled()
+                ? wc_get_page_screen_id( 'shop_order' )
+                : 'shop_order';
```

Comment:
We need to include a link and some description that this is "how to add meta boxes to the legacy order editor screen when legacy orders are in effect, and to the new HPOS-powered editor screen otherwise."

See: https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book#audit-for-order-administration-screen-functions

Do we really need the globals? It seems like we should refactor to a "render_smart_send_metabox" method as well: https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book#audit-for-order-administration-screen-functions

Reply:
Hi. In my last PR I worked on the following tasks:
I checked in your plugin code and changed next functions:
1)Auditing the code base for direct DB access usage - I checked all the functions and noted which are used in your plugin and which are not in this document (https://docs.google.com/spreadsheets/d/1pRkiBV7oMJhUf-oW95p33zqxJ0fNFhTQE1W1s-zD3pk/edit#gid=0)
2)APIs for getting/setting posts and postmeta - checked
3)Audit for order administration screen functions -started work

Debug log with: Detecting whether - HPOS tables are being used in the store- Do I need to do this check for all the old functions that I will be replacing with new ones for HPOS?

I didn’t work on your project today, I’ll take the whole day tomorrow.

Reply:
@AlexShevchenko1985 I moved your comment to the PR itself and answered there https://github.com/smartsendio/woocommerce/pull/41#issuecomment-1874132388

But please see my comment above about the globals and check if this is needed?